gui/mozregui/wizard.py

```diff
@@ -29,6 +29,15 @@ def resolve_obj_name(obj, name):
     return obj
 
 
+def changelabel(self, checkstatus):
```

Comment:
Could you make this a method in a class ? I think it would be easier to understand and better structured.
Maybe in a:

class WizardSelectionRangePage(WizardPage):
    def change_label()
        ...
    def initializePage()
        ...
Then you could inherit from it for inbound/Nightlies:

class InboundPage(WizardSelectionRangePage):
    ...
class NightliesPage(WizardSelectionRangePage):
    ...